src/data/chats/chat.rs

```diff
@@ -51,6 +51,8 @@ struct ChatData {
     title: String,
     #[serde(default)]
     title_state: TitleState,
+    #[serde(default)]
```

Comment:
chrono will pick the unix epoch as the default, and that's important for this to work in a backward compatible way.

Reply:
We're using different ways of handling dates now, one for setting the id and another one for this, can we unify it?